ups-connector-test/pom.xml

```diff
@@ -2,15 +2,15 @@
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>ups.connector.test</groupId>
+  <groupId>com.axonivy.connector.ups</groupId>
   <artifactId>ups-connector-test</artifactId>
   <version>1.0.0-SNAPSHOT</version>
   <packaging>iar</packaging>
   <dependencies>
     <dependency>
-      <groupId>ups.connector</groupId>
+      <groupId>com.axonivy.connector.ups</groupId>
       <artifactId>ups-connector</artifactId>
-      <version>1.0.0</version>
+      <version>1.0.0-SNAPSHOT</version>
```

Comment:
please make it consistent with the version you defined in pom of ups-connector-product (10.0.14-SNAPSHOT)